internal_worker_base.go

```diff
@@ -169,10 +175,14 @@ func (bw *baseWorker) execute(routineID int) {
 				if err == nil {
 					bw.retrier.Succeeded()
 				} else if isClientSideError(err) {
-					bw.logger.Info("Poll and processing failed with client side error", zap.Int(tagRoutineID, routineID), zap.Error(err))
+					traceLog(func() {
```

Comment:
This is too verbose, All poll errors are temporary issues, they are not real errors, since we have enough retires. This is one of the main reasons to hide these logs under verbose level.
if something at poll is important it should be logged as bw.logger.Error(...)